app/javascript/dashboard/i18n/locale/en/helpCenter.json

```diff
@@ -363,9 +363,9 @@
         "PORTAL": "Portal",
         "LOCALE": "Locale",
         "NAME": {
-          "LABEL": "Name",
+          "LABEL": "Icon & Name",
```

Comment:
We don't need to say "Icon & Name" explicitly, we can simply say "Name" the icon part is implied given that the picker is available

CC: @Hricha-Shandily @pranavrajs

Reply:
Moved back to Name

Reply:
Name is fine.